wrong number of type arguments

I hit this when upgrading from ffc74a17 to 8bda0d64. I haven't had time to dig in yet, (will try and set aside some time tomorrow if you haven't magically fixed it by then), but I thought I'd give you a quick heads up (i.e. hold off on publishing the updates to crates.io).

```
error[E0244]: wrong number of type arguments: expected 1, found 2
   --> src/templates/mod.rs:139:10
    |
139 | #[derive(Template)]
    |          ^^^^^^^^ expected 1 type argument
```

--- a/askama_derive/src/generator.rs
+++ b/askama_derive/src/generator.rs
@@ -368,7 +368,7 @@ impl<'a> Generator<'a> {
         self.writeln("#[allow(unused_variables)]");
         self.writeln(&format!(
             "fn render_block_{}_into(&self, writer: &mut ::std::fmt::Write) \
-             -> Result<(), ::std::fmt::Error> {{",
+             -> ::std::result::Result<(), ::std::fmt::Error> {{",
             name));
         self.prepare_ws(ws1);
         self.handle(nodes);
@@ -470,7 +470,7 @@ impl<'a> Generator<'a> {
     fn impl_template(&mut self, ast: &syn::DeriveInput, nodes: &'a [Node]) {
         self.write_header(ast, "::askama::Template");
         self.writeln("fn render_into(&self, writer: &mut ::std::fmt::Write) -> \
-                      Result<(), ::std::fmt::Error> {");
+                      ::std::result::Result<(), ::std::fmt::Error> {");
         self.handle(nodes);
         self.flush_ws(&WS(false, false));
         self.writeln("Ok(())");
@@ -481,7 +481,7 @@ impl<'a> Generator<'a> {
     // Implement `Display` for the given context struct.
     fn impl_display(&mut self, ast: &syn::DeriveInput) {
         self.write_header(ast, "::std::fmt::Display");
-        self.writeln("fn fmt(&self, f: &mut ::std::fmt::Formatter) -> Result<(), ::std::fmt::Error> {");
+        self.writeln("fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::result::Result<(), ::std::fmt::Error> {");
         self.writeln("self.render_into(f)");
         self.writeln("}");
         self.writeln("}");
@@ -508,7 +508,7 @@ impl<'a> Generator<'a> {
         self.writeln("#[allow(unused_variables)]");
         self.writeln(&format!(
             "fn render_trait_into(&self, timpl: &{}, writer: &mut ::std::fmt::Write) \
-             -> Result<(), ::std::fmt::Error> {{",
+             -> ::std::result::Result<(), ::std::fmt::Error> {{",
             trait_name));
 
         if let Some(nodes) = nodes {
@@ -528,7 +528,7 @@ impl<'a> Generator<'a> {
     fn impl_template_for_trait(&mut self, ast: &syn::DeriveInput, derived: bool) {
         self.write_header(ast, "::askama::Template");
         self.writeln("fn render_into(&self, writer: &mut ::std::fmt::Write) \
-                      -> Result<(), ::std::fmt::Error> {");
+                      -> ::std::result::Result<(), ::std::fmt::Error> {");
         if derived {
             self.writeln("self._parent.render_trait_into(self, writer)?;");
         } else {
@@ -546,12 +546,12 @@ impl<'a> Generator<'a> {
         for bname in block_names {
             self.writeln(&format!(
                 "fn render_block_{}_into(&self, writer: &mut ::std::fmt::Write) \
-                -> Result<(), ::std::fmt::Error>;",
+                -> ::std::result::Result<(), ::std::fmt::Error>;",
                 bname));
         }
         self.writeln(&format!(
             "fn render_trait_into(&self, timpl: &{}, writer: &mut ::std::fmt::Write) \
-             -> Result<(), ::std::fmt::Error>;",
+             -> ::std::result::Result<(), ::std::fmt::Error>;",
             trait_name));
         self.writeln("}");
     }

